vuepress/docs/next/tutorials/devops/external-id-management.md

```diff
@@ -2,75 +2,83 @@
 
 ## Purpose
 
-This tutorial enables the Entando administrator to leverage an existing
-Keycloak instance.
+This tutorial details how Entando can be connected to an existing Keycloak instance.
 
-## Requirements
+## Prerequisites
 
 -   A Keycloak instance
-
--   A realm named "entando" on that instance
-
--   An admin user for the "entando" realm
+-   A realm named "entando" in that instance
+-   Admin user credentials for the "entando" realm
 
 ## Steps
 
-### 1. Get keycloak information
-
-Retrieve the relevant information from the Keycloak instance you want to
-use.
-
-Specifically you will need:
+### 1. Get Keycloak information
 
--   The username of the Keycloak admin that has admin rights to the
-    "entando" realm, e.g entando-keycloak-admin
+Retrieve the following information from the existing Keycloak instance:
 
+-   The username of the Keycloak admin with rights to the "entando" realm, e.g entando-keycloak-admin
 -   The Keycloak admin password, e.g. password123
+-   The base URL for the Keycloak server, including the auth value, e.g. https://YOUR-KEYCLOAK-INSTANCE.com/auth
 
--   The base url for the Keycloak server, including the auth value, e.g.
-    https://your-keycloak-instance.com/auth
+> **Note** When connecting an external Keycloak instance to Entando, it is best practice to exclusively enter the admin credentials of a service account dedicated to the management and tracking of system changes.
```

Comment:
I'm not sure this has quite the meaning. Typically, you want to use a dedicated service account when connecting two systems, rather than an individual user's account. That way if the credentials change or you need to know who made a change you can tell the difference between the activities.  Maybe just "... it is a good practice to provide the credentials for a dedicated service account."